infrastructure/cli/cmd/create/helm.go

```diff
@@ -0,0 +1,218 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	watch "k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"log"
+)
+
+const airyYamlConfigMap = "airy-yaml-config"
+const serviceAccountName = "helm-account"
+
+const defaultAiryConfig = `
+global:
+  appImageTag: develop
+  containerRegistry: ghcr.io/airyhq
+  namespace: default
+`
```

Comment:
I get it was a little easier to do it this way but I'm not so comfortable making such an important concept just a string somewhere. As I understand it, it's also not so much work as now go supports embededding data into a binary natively (long overdue feature, https://golang.org/pkg/embed/ this should get us going).

I suggest we introduce `infrastructure/defaults.yml` (we can debate our guidelines about the configuration in a different pr, probably the best chance is doing that right before we eliminate the init command) and embed the file here

Reply:
We tried to go that route briefly in our pairing session, but noticed a few pitfalls right away, because of Gazelle. I would recommend that we tackle this in [this issue](https://github.com/airyhq/airy/issues/1051#issuecomment-788041906), which can serve as a follow-up where we also embed an entire directory and not just the Airy yaml

Reply:
while I understand the argument and we'd like to move forward with this:

- it's not ready anyway
- before we get to that issue it's going to be a while (as we have two providers between this and the command) 

And I'm afraid it's too much of an important concept to hide it even for a week or so. to be fair the argument "we tried and it seems tricky" works more in favor of doing it right away than waiting the whole codebase has more code.  

Now having said all of it, my opinion is not so strong I'd prevent us fro merging this (that's something I tend to do only with the "public api" and the docs). So if the argument is not convincing, happy to add one more step to the issue you mention

Reply:
I changed the code so it uses a `infrastructure/defaults.yaml` as @lucapette mentioned